backend/src/main/java/xbdd/webapp/resource/feature/User.java

```diff
@@ -35,46 +37,42 @@ public User(final MongoDBAccessor client) {
 
 	@GET
 	@Path("/tagAssignment/{product}/{major}.{minor}.{servicePack}/{build}")
-	@Consumes("application/json")
-	public DBObject getTagsAssignment(@BeanParam final Coordinates coordinates) {
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response getTagsAssignment(@BeanParam final Coordinates coordinates) {
 		final DB db = this.client.getDB("bdd");
 		final DBCollection collection = db.getCollection("tagsAssignment");
 		final BasicDBObject coq = coordinates.getReportCoordinatesQueryObject();
 		final DBObject document = collection.findOne(coq);
 
 		if (document != null) {
 			final BasicDBList tags = (BasicDBList) document.get("tags");
-			return tags;
+			return Response.ok(SerializerUtil.serialise(tags)).build();
 		}
-		return null;
+
+		return Response.noContent().build();
 	}
```

Comment:
These functions which return a list would be way nicer to use if they returned an empty list rather than null / no content. That way the front end doesn't have to have optional fields because it will always be there.
I saw you just raised an issue that scenario tags can be null - changing the backend to return an empty list instead of null would solve that issue.

Reply:
Yeah thats a much better idea, I will update that.

Reply:
Were you going to do this change in this branch or separately?

Reply:
I will do it now